src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -61,6 +88,15 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
     Utils utils;
     vector<Utils::worker> workerList = utils.getWorkerList(sqlite);
     vector<DataPublisher*> workerClients;
+//  Initiate Kafka consumer
+    thread th1;
+    cppkafka::Configuration configs = {{"metadata.broker.list", "127.0.0.1:9092"},
```

Comment:
Why do we have the IP address and the port hard coded here?

Reply:
For the testing purpose I hard code it. Now I loaded it from the jasminegraph-server.properties file.